hackathon_site/dashboard/frontend/src/components/orders/OrdersCount/OrdersCount.tsx

```diff
@@ -1,12 +1,18 @@
 import React from "react";
 import { IconButton, Typography } from "@material-ui/core";
 import RefreshIcon from "@material-ui/icons/Refresh";
-import { useSelector } from "react-redux";
-import { FormikValues } from "formik";
+import { useDispatch, useSelector } from "react-redux";
 import styles from "pages/Orders/Orders.module.scss";
-import { adminOrderTotalSelector } from "../../../slices/order/adminOrderSlice";
+import {
+    adminOrderTotalSelector,
+    getOrdersWithFilters,
+} from "../../../slices/order/adminOrderSlice";
```

Comment:
We use absolute imports(basically we don't want the "...", so you can replace it with `slices/order/adminOrderSlice`